sources/north-america/us/oh/ODOT_TIMS_Road_Inventory.geojson

```diff
@@ -0,0 +1,1488 @@
+{
+  "type": "Feature",
+  "properties": {
+    "id": "ODOT_TIMS_Road_Inventory",
+    "name": "ODOT Road Inventory",
+    "description": "Ohio Department of Transportation (ODOT) - Transportation Information Mapping System (TIMS) Road Inventory published June 2024.",
+    "type": "tms",
+    "url": "https://tiles.mblaine.com/ohio/{zoom}/{x}/{y}.png",
+    "min_zoom": 0,
+    "max_zoom": 19,
+    "license_url": "https://wiki.openstreetmap.org/wiki/ODOT_TIMS#Right_to_Use",
+    "privacy_policy_url": "https://www.transportation.ohio.gov/help-center/privacy-notice-and-policies",
+    "permission_osm": "implicit",
+    "icon": "https://www.transportation.ohio.gov/wps/wcm/connect/gov/2ac66df0-55c6-4360-9f44-ff1435bbd068/favicon-ODOT.png?MOD=AJPERES",
+    "i18n": true,
+    "country_code": "US",
+    "attribution": {
+      "url": "https://gis.dot.state.oh.us/tims_classic/About",

```

Comment:
I’m a bit surprised ODOT no longer has a full page describing TIMS in more detail. [This page on the main ODOT site](https://www.transportation.ohio.gov/working/data-tools/resources/03-tims) doesn’t say a whole lot more. 🤷‍♂️

Reply:
Yeah I haven't been able to find a better explanation of the whole thing. This [Roadway Information Manual](https://www.transportation.ohio.gov/programs/technical-services/tech-services-card-catalog/roadway-manual) describes the road inventory specifically.

I did find this, referencing a new version on the website they're working on (page 11): https://www.bceo.org/wp-content/uploads/2024/06/1030b-TIMS_Data_Update_TEW2024.pdf